contracts/DCAKeep3rJob/DCAKeep3rJob.sol

```diff
@@ -13,13 +13,8 @@ contract DCAKeep3rJob is Governable, IDCAKeep3rJob {
   address public swapper;
   mapping(address => bool) public canAddressSignWork;
 
-  constructor(
-    address _swapper,
-    IKeep3rJobs _keep3r,
-    address _governor
-  ) Governable(_governor) {
-    if (address(_swapper) == address(0) || address(_keep3r) == address(0)) revert ZeroAddress();
-    swapper = _swapper;
+  constructor(IKeep3rJobs _keep3r, address _governor) Governable(_governor) {
+    if (address(_keep3r) == address(0)) revert ZeroAddress();
```

Comment:
I would keep the parameter, but allow the zero address. So that it can be set when deployed, but it's just not necessary. Otherwise you'll be forzing an extra tx always